### ISSUE TITLE
Bookmarks don't work on 88.0.4324.95

### ISSUE THREAD

**Issue by lowtrucks** [bromite/bromite#891]
### Bromite version
Version: `88.0.4324.95`
Arch: `arm`
Android version: `9.0`
Device model: `nx569j`

### Is this bug about the SystemWebView?
No.

### Is the bug reproducible with latest version?
Yes
.
### Can the bug be reproduced with corresponding Chromium version?
I don't know.

### Is the bug a crash?
No,

### Describe the bug
Bookmarks don't work on 88.0.4324.95.

### Steps to reproduce the bug
1. Open any bookmark.
2. Nothing happens - it just redirects to empty Mobile Bookmarks.

### Expected behavior
When you click on a bookmark, it opens a site in bookmark.

**Comment by csagan5** [comment]
It is missing the Chromium test and I cannot reproduce it.

**Comment by BlueMax** [comment]
As a workaround for now one can hold/select a bookmark entry, go to 3-dot menu and select 'open in new tab'.

**Comment by drk002** [comment]
I have the same. Selecting from bookmarks does not work. When I select an item from the tab, the screen blinks and nothing happens.

**Comment by ajay230ps** [comment]
Same issues with me too. No issues found in the chromium build. I'm able to open website from the bookmark. 
Chromium Build: 88.0.4324.95
Device: tissot
Android: 10
Arc: arm64
Bromite Build: 88.0.4324.95

**Comment by uazo** [comment]
I'm comparing your version 88 with what I did and in my version it works.
in the Add-bookmark-import-export-actions.patch I notice this lack

```
diff --git a/chrome/android/java/src/org/chromium/chrome/browser/native_page/NativePageFactory.java b/chrome/android/java/src/org/chromium/chrome/browser/native_page/NativePageFactory.java
--- a/chrome/android/java/src/org/chromium/chrome/browser/native_page/NativePageFactory.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/native_page/NativePageFactory.java
@@ -95,7 +95,7 @@ public class NativePageFactory {
 
         protected NativePage buildBookmarksPage(Tab tab) {
             return new BookmarkPage(mActivity.getComponentName(), mActivity.getSnackbarManager(),
-                    new TabShim(tab, mActivity));
+                    new TabShim(tab, mActivity), mActivity);
         }
 
         protected NativePage buildDownloadsPage(Tab tab) {
```

here https://gist.github.com/uazo/5103fb3cf38f813f6d5e23acb58ca1c2 the complete patch,
i'm still checking the other patches because i used a different approach, after applying every single patch i always recompiled, i let you know.

**Comment by tp0** [comment]
> As a workaround for now one can hold/select a bookmark entry, go to 3-dot menu and select 'open in new tab'.

Also one can access bookmarks via `chrome://bookmarks` which works normally.

**Comment by Xeno-zombi** [comment]
Usually Bromite does a good job fixing bugs quickly. So why is the bookmark issue still not fixed?

**Comment by claytoncarney** [comment]
I experienced the same problems and observed the workaround mentioned. I tried clearing cache, then storage: to no avail. The only way I found to eliminate the issue was to uninstall/reinstall the app.

**Comment by drk002** [comment]
> 
> 
> Usually Bromite does a good job fixing bugs quickly. So why is the bookmark issue still not fixed?

I am also very surprised by this. After all, it's a basic browser feature. With this error, the browser is useless.

**Comment by MTSYuki** [comment]
Same issue. After a fresh install, I just copied all of my bookmarks from Fennec. All successfully saved, but I noticed the same problem when I clicked them.

Version: 88.0.4324.95

Device: LG-M250 Build/NRD90U

Android: 7.0

Arch: armv7

**Comment by csagan5** [comment]
Fixed in `88.0.4324.141`.

**Comment by JVS-XII** [comment]
Thanks, I was already thinking of doing reset😁
